_benchmark/reference/commands/run.md

```diff
@@ -86,11 +87,11 @@ After the test runs, OpenSearch Benchmark responds with a summary of the benchma
 
 ## Options
 
-Use the following options to customize the `execute-test` command for your use case. Options in this section are categorized by their use case. 
+Use the following options to customize the `run` command for your use case. Options in this section are categorized by their use case. 
 
 ## General settings
 
-The following options shape how each test runs and how results appear: 
+The following options customize each run and affects how run results appear: 
```

Comment:
I think we can reword this to be --> ...customize each test run and affects how results appear